@kiva/kv-components/src/vue/KvSideSheet.vue

```diff
@@ -195,6 +195,13 @@ export default {
 				return false;
 			},
 		},
+		/**
+		 * Whether to hide the background of the headline section on mobile (e.g., in Borrower Profile Sidesheet)
+		 * */
+		hideHeadlineBgOnMobile: {
```

Comment:
Why is this passed in as a flag? Are there different use cases for mobile?

Reply:
Where is it hidden? Just trying to understand the differences between the use cases, but I thought they were relatively the same.

Reply:
Ah makes sense, thanks for the example!